src/main/java/com/zufar/onlinestore/user/exception/UserNotFoundException.java

```diff
@@ -0,0 +1,24 @@
+package com.zufar.onlinestore.user.exception;
+
+import lombok.Getter;
+
+import java.util.UUID;
+
+@Getter
+public class UserNotFoundException extends RuntimeException {
+
+    private final UUID userId;
+    private final String userName;
+
+    public UserNotFoundException(UUID userId) {
+        super(String.format("User with id = %s is not found.", userId));
```

Comment:
maybe just leave only one version of constructor with id?

Reply:
fixed